test/nut/commands/force/org/org.nut.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import * as querystring from 'querystring';
+import { expect } from '@salesforce/command/lib/test';
+import { TestSession } from '@salesforce/cli-plugins-testkit';
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { asDictionary, AnyJson, Dictionary, getString, isArray } from '@salesforce/ts-types';
+
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
If we disable rules we must add a comment explaining why.  Also, is it really worth disabling the rule if we can just write the code in a slightly different way?  Pretty sure this is so we can do: `expect(devHubLine).to.be.ok;` which is arguably unnecessary.

Reply:
@shetzel I will remove it and the `ok` checks